<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomDialog</class>
 <widget class="QDialog" name="RoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="FiveInLine" name="wdg_play" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>580</width>
     <height>580</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>580</width>
     <height>580</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>580</width>
     <height>580</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_icon_player1">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>100</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Resource2/icon/slotwait.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_icon_player2">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>250</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>101</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Resource2/icon/slotwait.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="player2layout">
    <item>
     <widget class="QLabel" name="lb_player2">
      <property name="minimumSize">
       <size>
        <width>138</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>138</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>玩家2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_player2_name">
      <property name="minimumSize">
       <size>
        <width>137</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="le_chat">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>552</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>231</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>231</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tb_chatShow">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>351</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>201</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>201</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_send">
   <property name="geometry">
    <rect>
     <x>842</x>
     <y>552</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ads">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>广告位出租／人◕ ‿‿ ◕人＼</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="player1layout">
    <item>
     <widget class="QLabel" name="lb_player1">
      <property name="minimumSize">
       <size>
        <width>138</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>138</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>玩家1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_player1_name">
      <property name="minimumSize">
       <size>
        <width>137</width>
        <height>39</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>137</width>
        <height>39</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>111</width>
     <height>109</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="player1pblayout">
    <item>
     <widget class="QPushButton" name="pb_start">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>开局</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_player1_ready">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>待准备</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_player1_cpu">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>托管</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>261</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="player2Layout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pb_player2_ready">
      <property name="minimumSize">
       <size>
        <width>109</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>109</width>
        <height>33</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>待准备</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb_player2_cpu">
      <property name="minimumSize">
       <size>
        <width>109</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>109</width>
        <height>33</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>托管</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FiveInLine</class>
   <extends>QWidget</extends>
   <header location="global">FiveInLine/fiveinline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
